plugins/destination/postgresql/client/client.go

```diff
@@ -24,11 +24,6 @@ type Client struct {
 	batchSize           int
 }
 
-type pgTablePrimaryKeys struct {
```

Comment:
I replaced this with a map in the one function where we were using it, seems a bit simpler. We can add this back later if we need it.